R/civis_future.R

```diff
@@ -0,0 +1,167 @@
+#' Evaluate an expression in Civis Platform
+#'
+#' @param ... Arguments to \code{\link{CivisFuture}}
+#' @return The result of evaluating \code{expr}.
+#'
+#' @examples \dontrun{
+#'  eval_civis(quote(sqrt(10)))
+#' }
+#'
+#' @export
+civis_platform <- function(...) {
+  future <- CivisFuture(...)
+  if (!future$lazy) future <- run(future)
+  future
+}
+class(civis_platform) <- c("CivisFuture", "future", "function")
+
+#' Evaluate an expression in Civis Platform
+#' @inheritParams future::Future
+#' @param required_resources resources
```

Comment:
Maybe mention these are the resources that get passed into `scripts_post_containers` (or wherever the more complete docs for `required_resources` live).